deployments/cardano.nix

```diff
@@ -43,28 +45,28 @@ let
   };
 
   regionIndex = region: keypair: testIndex: cardano-node { inherit region testIndex keypair; };
-  cardano-node-eu = regionIndex "eu-central-1" (pairs: pairs.cardano-test-eu);
-  cardano-node-eu_old = regionIndex "eu-central-1" (pairs: pairs.my-key-pair);
-  cardano-node-us = regionIndex "us-west-1" (pairs: pairs.cardano-test-us);
-  cardano-node-asia = regionIndex "ap-southeast-1" (pairs: pairs.cardano-test-asia);
-  cardano-node-sydney = regionIndex "ap-southeast-2" (pairs: pairs.cardano-test-sydney);
-  cardano-node-sa = regionIndex "sa-east-1" (pairs: pairs.cardano-test-sa);
-in 
+
+  cardano-node-eu-central = regionIndex "eu-central-1" (pairs: pairs.cardano-test-eu-central);
+  cardano-node-eu-west-1 = regionIndex "eu-west-1" (pairs: pairs.cardano-test-eu-west-1);
+  cardano-node-eu-west-2 = regionIndex "eu-west-2" (pairs: pairs.cardano-test-eu-west-2);
+  cardano-node-ap-southeast-1 = regionIndex "ap-southeast-1" (pairs: pairs.cardano-test-ap-southeast-1);
+  cardano-node-ap-southeast-2 = regionIndex "ap-southeast-2" (pairs: pairs.cardano-test-ap-southeast-2);
+  cardano-node-ap-northeast-1 = regionIndex "ap-northeast-1" (pairs: pairs.cardano-test-ap-northeast-1);
+  cardano-node-ap-northeast-2 = regionIndex "ap-northeast-2" (pairs: pairs.cardano-test-ap-northeast-2);
+
+in
   # NOTE: Also update resources at the bottom
-  (genAttrs' (range 1 5) (key: "node${toString key}") (name: cardano-node-eu name)) // 
-  #(genAttrs' (range 3 5) (key: "node${toString key}") (name: cardano-node-us name)) // 
-  #(genAttrs' (range 20 29) (key: "node${toString key}") (name: cardano-node-asia name)) //
-  #(genAttrs' (range 30 39) (key: "node${toString key}") (name: cardano-node-sydney name)) //
-  #(genAttrs' (range 40 49) (key: "node${toString key}") (name: cardano-node-sa name)) //
-  #(genAttrs' (range 50 59) (key: "node${toString key}") (name: cardano-node-eu name)) // 
-  #(genAttrs' (range 60 69) (key: "node${toString key}") (name: cardano-node-us name)) // 
-  #(genAttrs' (range 70 79) (key: "node${toString key}") (name: cardano-node-asia name)) //
-  #(genAttrs' (range 80 89) (key: "node${toString key}") (name: cardano-node-sydney name)) //
-  #(genAttrs' (range 90 99) (key: "node${toString key}") (name: cardano-node-sa name)) //
+  (genAttrs' (range 1 1) (key: "node${toString key}") (name: cardano-node-eu-central name)) //
```

Comment:
It's well hidden, but here is index 0: https://github.com/input-output-hk/iohk-nixops/pull/23/files#diff-0fba1cb62377263a973e715816efbc95R69